File: 프로그래머스/0/120825. 문자 반복 출력하기/문자 반복 출력하기.js
function solution(my_string, n) {
    var answer = '';
    
    for(let s = 0; s < my_string.length; s++)
    {
        for(let i = 0; i < n; i++)
        {
            answer += my_string[s];
        }
    }
    
    return answer;
}

File: 프로그래머스/0/120825. 문자 반복 출력하기/문자 반복 출력하기.java
class Solution {
    public String solution(String my_string, int n) {
        String answer = "";
        
        for(int s = 0; s < my_string.length(); s++)
        {
            for(int i = 0; i < n; i++)
            {
                answer += my_string.charAt(s);
            }
        }
        
        return answer;
    }
}